spec/models/certificate_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "rails_helper"
+
+describe Certificate, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Is this intended to be committed?

Reply:
there is not much to test now in this model but i could add a basic test, so we can extend it later if we need to. i'd rather not drop it